src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

```diff
@@ -116,6 +126,37 @@ public void itemStateChanged(ItemEvent e) {
 
         toolbar.add(Box.createHorizontalStrut(16));
         toolbar.add(showErrorsToggleButton);
+        
+        
+        JButton button = new JButton((Constant.messages.getString("fuzz.httpfuzzer.results.toolbar.button.export")));
+        button.setIcon(new ImageIcon(HttpFuzzResultsContentPanel.class.getResource("/resource/icon/16/115.png")));
+        button.addActionListener((new AbstractAction() {
+			private static final long serialVersionUID = 1L;
+
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				WritableFileChooser chooser = new WritableFileChooser();
+				chooser.setSelectedFile(new File(Constant.messages.getString("fuzz.httpfuzzer.results.toolbar.button.export.defaultName")));
+				if (chooser.showSaveDialog(null) == WritableFileChooser.APPROVE_OPTION) {
+					String file = chooser.getSelectedFile().toString();
+					if (!file.endsWith(".csv")) file += ".csv";
+					try (CSVPrinter pw = new CSVPrinter(new PrintWriter(new File(file)), CSVFormat.newFormat(',').withRecordSeparator("\n"))) {
+						pw.printRecord(currentFuzzer.getMessagesModel().getHeaders());
+						int count = currentFuzzer.getMessagesModel().getRowCount();
+		        		for (int i = 0; i < count; i++) {
+		        			pw.printRecord(currentFuzzer.getMessagesModel().getEntry(i).getFormat());
+		                }
+		        		JOptionPane.showMessageDialog(null, Constant.messages.getString("fuzz.httpfuzzer.results.toolbar.button.export.showMessageSuccessful"));

```

Comment:
The parent could be "View.getSingleton().getMainFrame()", or it could just use the View class to show the message.
Same comment for the error message (but show a warning).